chrome_extension/background/event.js

```diff
@@ -124,7 +116,7 @@ mooltipassEvent.onSaveSettings = function(callback, tab, settings) {
 	mooltipassEvent.onLoadSettings();
 }
 
-mooltipassEvent.onGetStatus = function(callback, tab) {
+mooltipassEvent.onGetStatus = function(callback, senderTab, tab) {
```

Comment:
I'll rephrase: any future content script running on a given tab will now have to pass as an arg its tab, correct? 
mooltipassEvent.onGetStatus will not use send or sender.tab from mooltipassEvent.onMessage .